crud.py

```diff
@@ -0,0 +1,52 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def get_all_authors(
+        db: Session,
+        skip: int = 0,
+        limit: int = 10
+):
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = models.Author(
+        name=author.name,
+        bio=author.bio,
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_author_by_id(db: Session, author_id: int):
+    return db.query(models.Author).filter(models.Author.id == author_id).first()
+
+
+def get_books(
+        db: Session,
+        skip: int = 0,
+        limit: int = 10
+):
+    return db.query(models.Book).offset(skip).limit(limit).all()
```

Comment:
Ensure that `models.Book` exists in the `models` module and has the necessary attributes like `id`, `title`, `summary`, `publication_date`, and `author_id`.